Chatkit.xcodeproj/project.pbxproj

```diff
@@ -1280,9 +1254,11 @@
 			buildRules = (
 			);
 			dependencies = (
-				73A9391823E0A7440025AF90 /* PBXTargetDependency */,
 			);
 			name = "Test Utilities";
+			packageProductDependencies = (
+				22B2D2BC23E2FEE00085D79C /* PusherChatkit */,
```

Comment:
Is there a reason that you’ve linked the `PusherChatkit` *package* to `Test Utililties` as opposed to just adding the `Chatkit` target as a dependency?  (It seems a little circular to have the final package linked back into one of the projects own targets)

<img width="588" alt="Screenshot 2020-01-31 at 12 34 26" src="https://user-images.githubusercontent.com/58689743/73541010-49c1b180-4429-11ea-9437-5ec38487fd12.png">

vs

<img width="518" alt="Screenshot 2020-01-31 at 12 34 38" src="https://user-images.githubusercontent.com/58689743/73541071-71187e80-4429-11ea-819a-3f50fa75d83b.png">

NOTE using the `dependency` option does require chaning build setting to `SWIFT_INSTALL_OBJC_HEADER` to `NO` to get things to compile.  Howeve since we aren't supporting objective-c I don't think this is an issue?

Reply:
It was because of `SWIFT_INSTALL_OBJC_HEADER` being set to `YES`. I will change that, but `Chatkit` should also be included in `Link Binary With Libraries` section, because `Test Utilities` code actually links to this framework.

Reply:
Thanks.  It still compiles without being linked but best to explicitly link it anyway 👍